src/eventbridge/send.ts

```diff
@@ -3,24 +3,25 @@ import { EventEntry } from './EventEntry';
 import { Entries } from './Entries';
 import { SendResponse } from './SendResponse';
 import logger from '../observability/logger';
+import { Differences } from '../utils/differences';
 import { TestActivity } from '../utils/testActivity';
+import { EventType } from '../utils/eventType';
 
 const eventbridge = new EventBridge();
-const sendEvents = async (testResults: TestActivity[]): Promise<SendResponse> => {
+const sendEvents = async (events: Array<Differences | TestActivity>, type: EventType): Promise<SendResponse> => {
   logger.info('sendEvents starting');
-  logger.info(`${testResults.length} ${testResults.length === 1 ? 'event' : 'events'} ready to send to eventbridge.`);
+  logger.info(`${events.length} ${events.length === 1 ? 'event' : 'events'} ready to send to eventbridge.`);
 
   const sendResponse: SendResponse = {
     SuccessCount: 0,
     FailCount: 0,
   };
 
-  for (let i = 0; i < testResults.length; i++) {
-    const result = testResults[i];
+  // eslint-disable-next-line no-restricted-syntax
+  for (const event of events) {
     const entry: EventEntry = {
       Source: process.env.AWS_EVENT_BUS_SOURCE,
-      // eslint-disable-next-line security/detect-object-injection
-      Detail: `{ "testResult": "${JSON.stringify(result)?.replace(/"/g, '\\"')}" }`,
+      Detail: `{ "testResult": "${JSON.stringify(event)?.replace(/"/g, '\\"')}" }, type: ${type}`,
```

Comment:
I _think_ that `type:` might be outside the `Detail` object here, and that's not clear as it's on the same line... byt that I mean I think this would produce the following object:

```
{
  "Source": "__AWS_EVENT_BUS_SOURCE__",
  "Detail": {
    "testResult": "__DOUBLE_QUOTE_ESCAPED_JSON_STRING__"
  },
  type: ${type}`,
  "DetailType": 'CVS ATF Test Result',
  "EventBusName": __AWS_EVENT_BUS_NAME__,
  "Time": 1665072854283,
}
```

When I think what's intended is:
```
{
  "Source": "__AWS_EVENT_BUS_SOURCE__",
  "Detail": {
    "testResult": "__DOUBLE_QUOTE_ESCAPED_JSON_STRING__",
    "type":" "${type}"
  },
  "DetailType": 'CVS ATF Test Result',
  "EventBusName": __AWS_EVENT_BUS_NAME__,
  "Time": 1665072854283,
}
```

```suggestion
      Detail: `{ "testResult": "${JSON.stringify(event)?.replace(/"/g, '\\"')}", "type": "${type}" }`,
```

Reply:
Good spot, you should be able to test your lambda on a feature env against Dynamics QA/Test environment (as that is where our feature lambdas default to) to see if it works as expected.

Reply:
Yes, would be a really good idea to test on a feature branch first. Is 'type' the reason for the billing amendment?

Reply:
Looks like 'type' goes in it's own element.
`{
"dvsa_message": "<message payload>",
"dvsa_type": 510720001
}`
Where it is now is fine, but the eventbridge transformer will need amending.

Reply:
Testing it in a feature environment is definitely the way to go